docs/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+TableTransforms = "0d432bfd-3ee1-4ac1-886a-39f05cc69a3e"
```

Comment:
We also need to fix the Documenter version to avoid breaking changes. You can manually add a compat section, for example: 

```julia
[compat]
Documenter = "0.27"
```

Reply:
OK

Reply:
Made in [040f279](https://github.com/JuliaML/TableTransforms.jl/pull/46/commits/040f2798f0f38750a5964f75bb7978fed2324465)